parrots.json

```diff
@@ -683,5 +683,9 @@
     {
       "gif": "braveheartparrot.gif",
       "name": "Braveheart Parrot"
+    },
+    {
+      "gif": "hd/maracasparrot.gif",
```

Comment:
Whoops!  I missed this on the first pass, sorry.  It should be `"hd": "hd/maracasparrot.gif",` . Sorry!

Reply:
Ah I was wondering why it was failing all checks lol,

cheers